.github/workflows/builder.yml

```
@@ -0,0 +1,313 @@
+name: Build images
+
+# yamllint disable-line rule:truthy
+on:
+  release:
+    types: ["published"]
+  schedule:
+    - cron: "0 2 * * *"
+
+env:
+  BUILD_TYPE: core
+  DEFAULT_PYTHON: 3.8
+
+jobs:
+  init:
+    name: Initialize build
+    runs-on: ubuntu-latest
+    outputs:
+      architectures: ${{ steps.info.outputs.architectures }}
+      version: ${{ steps.version.outputs.version }}
+      channel: ${{ steps.version.outputs.channel }}
+      publish: ${{ steps.version.outputs.publish }}
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+
+      - name: Get information
+        id: info
+        uses: home-assistant/actions/helpers/info@master
+
+      - name: Get version
+        id: version
+        uses: home-assistant/actions/helpers/version@master
+        with:
+          type: ${{ env.BUILD_TYPE }}
+
+      - name: Verify version
+        uses: home-assistant/actions/helpers/verify-version@master
+        with:
+          ignore-dev: true
+
+  build_python:
+    name: Build PyPi package
+    needs: init
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        if: needs.init.outputs.publish == "true"
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
+        if: needs.init.outputs.publish == "true"
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+
+      - name: Build package
+        if: needs.init.outputs.publish == "true"
+        shell: bash
+        run: |
+          pip install twine wheel
+          python setup.py sdist bdist_wheel
+
+      - name: Upload package
+        if: needs.init.outputs.publish == "true"
+        shell: bash
+        run: |
+          export TWINE_USERNAME="$(twineUser)"
+          export TWINE_PASSWORD="$(twinePassword)"
```

We should use a specific token here:

```suggestion
          export TWINE_USERNAME="__token__"
          export TWINE_PASSWORD="$(twinePassword)"
```

Where does twinePassword come from? Should this have been TWINE_TOKEN instead?